launch

```diff
@@ -198,16 +198,26 @@ def install_mongodb():
   elif (os_name == 'Ubuntu') and (os_dist in supported):
     Log.info('Install mongodb with "apt-get" (sudo privileges required)')
     cmds = ['sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80' \
-      + ' --recv 0C49F3730359A14518585931BC711F9BA15703C6', '', \
+      + ' --recv 9DA31620334BD75D9DCB49F368818C72E52529D4', '', \
       'sudo apt-get update', 'sudo apt-get install -y mongodb-org']
+    #Ubuntu 14.04
     if os_dist == 'trusty':
       cmds[1] = 'echo "deb [ arch=amd64 ] http://repo.mongodb.org/apt/ubuntu' \
         + ' trusty/mongodb-org/3.4 multiverse"' \
         + ' | sudo tee /etc/apt/sources.list.d/mongodb-org-3.4.list'
-    else:
+    # Ubuntu 16.04
+    elif os_dist == 'xenial':
       cmds[1] = 'echo "deb [ arch=amd64,arm64 ] http://repo.mongodb.org/apt/' \
         + 'ubuntu xenial/mongodb-org/3.4 multiverse"' \
         + ' | sudo tee /etc/apt/sources.list.d/mongodb-org-3.4.list'
+    # Ubutnu 18.04
+    elif os_dist == 'bionic':
+      cmds[1] = 'echo "deb [ arch=amd64,arm64 ] http://repo.mongodb.org/apt/ubuntu' \
+        + ' bionic/mongodb-org/4.0 multiverse"' \
```

Comment:
Please use the same version for all distributions for consistency. Also, if we are going to use 4.0, docker image mongod must be bumped as well.

Reply:
there are no MongoDB 4.0 packages for trusty.
There are no MongoDB 3.4 packages for bionic
So the only option which I see to keep consistency is to support only 
xenial and bionic with mongodb 4.0...

Reply:
ok